### PR TITLE
Leveling Colliding Events

### PR DESCRIPTION
**Pull request by JFickel**

### CODE REVIEW

File: tests/unit/models/event-list-test.js [hunk 1]
```diff
@@ -46,3 +46,21 @@ test('EventList#forDay returns the correct events for multi-day events', functio
 
   assert.ok(true);
 });
+
+test('EventList#setUpLevelsForDay sets the correct levels on events', function(assert) {
```
Comment (JFickel) [inline tests/unit/models/event-list-test.js]: Oops. Changed the name of the method.

File: addon/components/calendar-event.js [hunk 2]
```diff
@@ -7,15 +7,29 @@ export default Ember.Component.extend({
   event: null,
   classNames: ['event'],
   attributeBindings: ['style'],
+  cluster: null,
```
Comment (ebryn) [inline addon/components/calendar-event.js]: embed the cluster inside the event

File: addon/models/event-list.js [hunk 3]
```diff
@@ -32,6 +32,54 @@ export default Ember.Object.extend({
     return lookup;
   }),
 
+  getClusterForDay(day) {
+    let dateKey = moment(day.get('date')).format('YYYY-MM-DD');
```
Comment (ebryn) [inline addon/models/event-list.js]: would prefer this to be a wrapped up in a method on the day

File: tests/dummy/app/templates/weekly.hbs [hunk 4]
```diff
@@ -18,7 +18,7 @@
     <ul class="week-day">
       <li class="event-column">
         {{#each day.events as |event|}}
-          {{calendar-event event=event}}
+          {{calendar-event event=event cluster=day.eventCluster}}
```
Comment (ebryn) [inline tests/dummy/app/templates/weekly.hbs]: as mentioned above, avoid passing it in, get it via `event.cluster`

**Pull request merged**